src/openloco/config.h

```diff
@@ -7,12 +7,27 @@ namespace openloco::config
 {
 #pragma pack(push, 1)
 
+    enum flags
+    {
+        SHOW_HEIGHT_AS_UNITS = (1 << 1),
+    };
+
+    enum measurement_formats
+    {
+        FORMAT_IMPERIAL = 0,
+        FORMAT_METRIC = 1,
+    };
+
     struct config_t
     {
-        uint32_t unk_00;
-        uint16_t resolution_width;
-        uint16_t resolution_height;
-        uint8_t pad_06[526];
+        uint32_t flags;             // 0x00 (dword_50AEB4)
```

Comment:
The `dword_50AEB4` comments seem a bit too IDA-specific to me. What about `// 0x50AEB4, 0x00`?

Reply:
Sure, that's fine with me. I left the `dword` and all because they indicated sizes, but that can be inferred from the types, too, of course.